nevergrad/benchmark/plotting.py

```diff
@@ -655,7 +655,9 @@ def winrates_from_selection(
         data = np.array(winrates.iloc[:num_rows, : len(sorted_names)])
         # pylint: disable=anomalous-backslash-in-string
         best_names = [
-            (f"{name} ({100 * val:2.1f}%)").replace("Search", "")
+            (
+                f"{name} ({100 * val:2.1f}% +- {100 * np.sqrt(val*(1-val)/int(2 * victories.loc[name, name])):2.1f}/4)"
```

Comment:
why "/4" outside?

Reply:
changed.
happy to discuss offline, this is philosophically important :-)